hypothesis-python/src/hypothesis/internal/conjecture/junkdrawer.py

```diff
@@ -421,6 +421,10 @@ def find(self, condition: Callable[[T], bool]) -> T:
 _gc_start = 0
 _gc_cumulative_time = 0
 
+# Since gc_callback potentially runs in test context, and perf_counter
+# might be monkeypatched, we store a reference to the real one.
+_perf_counter = time.perf_counter
```

Comment:
`from time import perf_counter`  would of course be possible, but I think it's wise to make the purpose very explicit by an explicit assignment.

On module level, because we want to patch it in our self-tests.